json2graphite.rb

```diff
@@ -333,4 +345,18 @@ def influx_metrics(data, timestamp, parent_key = nil)
   end
 end
 
+if ARGF.filename == '-'
```

Comment:
This is hanging because ruby always has the option to read from STDIN. So `-` will be present even if you are not piping anything in. It hangs in this repo because we do not pipe anything in. Perhaps we can gate reading from STDIN on a command line argument?